docs/podman-generate-systemd.1.md

```diff
@@ -4,16 +4,19 @@
 podman-generate-systemd- Generate Systemd Unit file
 
 ## SYNOPSIS
-**podman generate systemd** [*options*] *container*
+**podman generate systemd** [*options*] *container|pod*
 
 ## DESCRIPTION
-**podman generate systemd** will create a Systemd unit file that can be used to control a container.  The
-command will dynamically create the unit file and output it to stdout where it can be piped by the user
-to a file.  The options can be used to influence the results of the output as well.
+**podman generate systemd** will create a systemd unit file that can be used to control a container.
+By default, the command will print the content of the unit files on stdout.
 
 
 ## OPTIONS:
 
+**--files**, **-f**
+
+Generate files instead of printing on stdout. The files option is not supported on the remote client.
```

Comment:
Perhaps add something like `the files will be named random-string.service and will be created in the current working directory`?

Reply:
I like that, thanks! Done.